src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -449,6 +457,20 @@ public virtual VerifyResult VerifyStateIndependent(ProtocolSettings settings)
             return VerifyResult.Succeed;
         }
 
+        private bool VerifyConflicts(IEnumerable<Transaction> mempool)
```

Comment:
This check is different with `MemoryPool.CheckConflicts`.  This may cause inconsistent problems.